src/common/enums/podcast/index.ts

```diff
@@ -1,2 +1,3 @@
-export { PodcastPeriodicity } from './podcastPeriodicity'
-export { PodcastType } from './podcastType'
+export { default as PodcastType } from './podcastType'
```

Comment:
It's not good that we always change the way of export each other. Please do as we did. We use default export only for components and styles.